api/v1alpha1/localvolumediscoveryresult_types.go

```diff
@@ -50,17 +32,9 @@ type DiscoveredDevice struct {
 	Type DiscoveredDeviceType `json:"type"`
 	// Vendor of the discovered device
 	Vendor string `json:"vendor"`
-	// Serial number of the disk
-	Serial string `json:"serial"`
 	// Size of the discovered device
 	Size int64 `json:"size"`
-	// Property represents whether the device type is rotational or not
-	Property DeviceMechanicalProperty `json:"property"`
-	// FSType represents the filesystem available on the device
-	FSType string `json:"fstype"`
-	// Status defines whether the device is available for use or not
-	Status DeviceStatus `json:"status"`
-	// WWN defines the WWN value of the device. For multipath devices, this is mandatory
+	// WWN defines the WWN value of the device.
 	WWN string `json:"WWN"`
```

Comment:
Have you considered using a lower case naming to follow with the rest of the fields naming convention?